src/mame/drivers/midvunit.cpp

```diff
@@ -12,16 +12,20 @@
         * War Gods (1996)           [3 sets]
         * Off Road Challenge (1997) [5 sets]
 
-    Known bugs:
-        * textures for automatic/manual selection get overwritten in Cruis'n World
-        * rendering needs to be looked at a little more closely to fix some holes
-        * in Cruis'n World attract mode, right side of sky looks like it has wrapped
-        * Off Road Challenge has polygon sorting issues, among other problems
-        * Issues for the Wargods sets:
-           All sets report as Game Type: 452 (12/11/1995) [which is wrong for newer sets]
+    Todo:
+        * Cruis'n World's neutral gear texture corruption
+        * Both games will eventually freeze in attract mode
+        * Some skybox textures in Cruis'n world do not load correctly and leak.
+        * Cruis'n USA's rom test, car select, high score screen and ending sequence plays at double speed
+        * RS232 support for communications (requires serial support from tms32031.cpp)
 
 Known to exist but not dumped:
-    Off Road Challenge v1.00 (Mon 07-28-97)
+	*Cruis'n USA L2.0
+	*Cruis'n USA L1.2
+	*Cruis'n World L2.6 (update to the Hyperdrive kit L2.5)
+	*Cruis'n World L1.0
+	*Off Road Challenge v1.00 (Mon 07-28-97)
```

Comment:
Can you please post evidence for this.  There have been far too many of these kinds of changes that are poorly sourced and end up wrong.